vantiq-apps-development/docs/eng/Vantiq_ExtLifecycleManagement_SetupProcedure.md

```diff
@@ -302,7 +302,9 @@ Click the the Deploy button.
 
 ![DeployButton](../../imgs/lcm/image5.png)
 
-　Fill in the required fields and click the OK button.
+Fill in the required fields.
+If pulling the image from a private repository, enter `quay-secret`, the docker-registry secret name which is created in the previous step, into the [Image Pull Secret] field.
```

Comment:
```suggestion
In case you want to pull the image from a private repository, enter `quay-secret`, the docker-registry secret name which you have created in the previous step, into the [Image Pull Secret] field.
```